atsign-foundation/atsign.dev-3.0
# fix: Update description & image for metadata

closes #137


**- What I did**
Updated the description, added the image

**- How to verify it**
We need to deploy that and check on discord
 
Some info for changelog:
Update the description, add the image in metadata
